apps/passport-server/src/services/provingService.ts

```diff
@@ -52,7 +52,9 @@ export class ProvingService {
     RSATicketPCDPackage,
     EdDSAPCDPackage,
     EdDSATicketPCDPackage,
-    RSAImagePCDPackage
+    RSAImagePCDPackage,
+    ZKEdDSAEventTicketPCDPackage,
+    ZKEdDSATicketPCDPackage
```

Comment:
note: don't think proving service is used anywhere in our logic right now. but thanks for getting this in for posterity!

Reply:
That matches what I heard when I asked why the old ZKEdDSATicketPCDPackage was initialized but never referenced in this service.  I figure either the service will get deleted, and this code with it, or the service will get used someday in which case adding this line will avoid confusion.